js/utils/trumbowyg.js

```diff
@@ -0,0 +1,29 @@
+import $ from 'jquery';
+import 'trumbowyg';
+
+$.trumbowyg.svgPath = '../node_modules/trumbowyg/dist/ui/icons.svg';
+
+export const editorOptions = {
+  btns: [
+    ['formatting'],
+    ['bold', 'italic'],
+    ['link'],
+    ['insertImage'],
+    'btnGrp-justify',
+    'btnGrp-lists',
+  ],
+};
+
+export function installRichEditor(attachPoint) {
```

Comment:
Can we please pass in the options as an argument and default it to the constant declared above? It would be nice to give the user the option of providing different options.

Something like:

```
export function installRichEditor(attachPoint, editorOptions = editorOptions) {
```